src/flask/cli.py

```diff
@@ -1018,20 +1018,22 @@ def routes_command(sort: str, all_methods: bool) -> None:
         for rule in rules
     ]
 
-    headers = ("Endpoint", "Methods", "Rule")
+    headers = ("Sub Domain", "Endpoint", "Methods", "Rule")
     widths = (
+        max(len(rule.subdomain) for rule in rules),
         max(len(rule.endpoint) for rule in rules),
         max(len(methods) for methods in rule_methods),
         max(len(rule.rule) for rule in rules),
     )
     widths = [max(len(h), w) for h, w in zip(headers, widths)]
-    row = "{{0:<{0}}}  {{1:<{1}}}  {{2:<{2}}}".format(*widths)
-
+    row = "{{0:<{0}}}  {{1:<{1}}}  {{2:<{2}}} {{3:<{3}}}".format(*widths)
```

Comment:
Why just one space between the last two headers instead of two like before?

Reply:
This is my first contribution to opensource stuff. Thought flask would be a good start because I use it very often. Apparently I have to write tests. I don't know how to write tests for what I did.

Reply:
Look at the existing tests that now fail and adapt them.

Reply:
Specifically, you need to fix the existing tests, which are now failing because you changed the output.

Reply:
Oh thank you, and that will resolve that 12 checks failed issue?, cuz I don't even know what failed. It say Linux, Windows, Mac, etc..

Reply:
click one one of the tests (pick one of the linux ones) and check the output. you can also run the tests locally.